assets/prototype/domain/eventEditor/datesList/AddNewDateButton.js

```diff
@@ -1,21 +1,37 @@
-import { Button } from '@blueprintjs/core/lib/esm';
-import { AppToaster } from '../EventEditor';
+import { useState } from "@wordpress/element";
+import { Button } from "@blueprintjs/core/lib/esm";
+import AddNewDateModal from "./AddNewDateModal";
 
 const btnStyle = {
-	margin: '0 0 0 1rem',
+	margin: "0 0 0 1rem"
 };
 
-const AddNewDateButton = () => (
-	<Button
-		icon={ 'calendar' }
-		text={ 'Add New Date' }
-		onClick={ () => AppToaster.show( {
-			intent: 'success',
-			message: 'New Date Created'
-		} ) }
-		style={ btnStyle }
-		large
-	/>
-);
+const AddNewDateButton = () => {
+	const overlayProps = {
+		autoFocus: true,
+		canEscapeKeyClose: true,
+		canOutsideClickClose: true,
+		enforceFocus: true,
+		hasBackdrop: true,
+		usePortal: true,
+		useTallContent: false
+	};
```

Comment:
what's this for? doesn't appear to be used...

Reply:
yes, had cleaned this up